CMakeLists.txt

```diff
@@ -125,6 +129,31 @@ elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
 	include_directories(SYSTEM "include/support/osx")
 endif()
 
+# If we're on windows, do a check for what compiler we're using (so we can link the appropriate libraries)
+if(WIN32)
+    if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU") # We're dealing with MinGW etc
+        set(Boost_USE_STATIC_LIBS ON)
+        set(BOOST_INCLUDEDIR ${LIB_ROOT_DIR}/boost/boost)
+
+        # Do a check for system bitness
+        if(AMD64)
+            set(BOOST_LIBRARYDIR ${LIB_ROOT_DIR}/boost/lib/amd64/debug)
+        else()
+            set(BOOST_LIBRARYDIR ${LIB_ROOT_DIR}/boost/lib/x86/debug)
+        endif()
+
+        set(GLEW_LIBRARY ${LIB_ROOT_DIR}/GLEW/lib/libglew.a)
+        set(SDL2_LIBRARY ${LIB_ROOT_DIR}/SDL2/lib)
+        set(ZLIB_LIBRARY ${LIB_ROOT_DIR}/zlib/lib/zlib.a)
```

Comment:
Wouldn't it suffice to set `GLEW_ROOT_DIR` & friends so that the libraries' CMake-configuration files search the path?
At this point `LIB_ROOT_DIR` might become the same as `CMAKE_PREFIX_PATH`. Perhaps try setting this to your "lib"-directory on Windows.

Reply:
> At this point LIB_ROOT_DIR might become the same as CMAKE_PREFIX_PATH. Perhaps try setting this to your "lib"-directory on Windows.

I think this might be the correct way to go.